content/blog/digib0y/2018-02-02_gci_2017_wrap_up_report.md

```diff
@@ -0,0 +1,56 @@
++++
+type = "blog"
+title = "GCI 2017 Wrap Up Report"
+author = "digib0y"
+date = "2018-02-02 00:13:41+05:30"
+tags = ["haiku", "software","GCI","Google","students","Code-In","Google Code-In","GCI2016"]
++++
+
+
+<p>Like all good things Google Code-In 2017 comes to an end, this year the program attracted over 3529 students in total for Google Code-In and out of those about ⅓ of the students claimed at least one task for Haiku, with 255 students completing at least one Haiku task. Haiku is also proud of its history of getting selected each year in <a href="http://codein.withgoogle.com">Google’s Code-In</a>.
+
+<p>Just like the other 24 organizations, Haiku picked winners out of the 10 students who completed the most tasks for Haiku, finalists and grand prize winners were picked collectively by the diligent team of Haiku’s 20+ mentors. The grand prize winners will be flown to Northern California to visit Google’s headquarters, enjoy a day of adventure in San Francisco, and meet one or two of their mentors and Google engineers.</p>
+
+<h3>Haiku’s Grand Prize Winners</h3>
+<ul>
+<li>
+Owen Pan from United States</li>
+<li>Hiếu Đức Lê from Vietnam</li>
+</ul>
+
+<h3>Haiku’s Finalists</h3>
+<ul>
+<li>Đắc Tùng Dương </li>
+<li>Xiang Fan </li>
+<li>Duong Dac</li>
+</ul>
+
+<h3>Rest of Haiku’s top ten finishers</h3>
+<ul>
+<li>Ng Young Shung</li>
+<li>Damillora </li>
+<li>Trần Thái Sơn </li>
+<li>Max Levchuk </li>
+<li>Mochammad Nur Afandi</li>
+</ul>
+
+<p>Congratulations everyone! Safe travels, grand prize winners! We are thankful for your precious contributions and will be delighted see you continue to <a href="https://www.haiku-os.org/community/getting-involved/">contribute</a> even after the program.</p>
+
+<p>Haiku had an impressive growth in the number of students:</p>
+
+<p><img src="/files/blog/digib0y/graph_gci2017.PNG" alt="Number of students" title=""></p>
```

Comment:
What is this graph supposed to show? It's a bit confusing because that 1106 number represents all the students who ever interacted with a Haiku task this year. The 3529 for overall GCI is for students who completed at least one task, according to the released statistics. I don't know if it's worth it to modify it now, but the number of all the students who interacted with tasks (even though they didn't solve any), is much bigger than that (and no org knows it obviously). Maybe we're looking for a graph showing the number of completed tasks per year? + Based on the last news the number is a bit higher: "It was a very, very busy 7 weeks for everyone - we had 3,555 students from 78 countries completing 16,468 tasks with a record 25 open source organizations!".

Reply:
I will make a new graph, in total we have 255 students completing at least one task for Haiku and 3,555 in total for GCI, is that right?